### DISCUSSION TITLE
测试及收集意见

### DISCUSSION THREAD

**Discussion by sikouhjw** [sikouhjw/gdutthesis#39] [General]
由于 Overleaf 还不支持 texlive 2021，我就改了一些设置以适应 texlive2020
Overleaf 版：https://cn.overleaf.com/2387673938nynfqfjzhvqt
texlive 2021 版：https://github.com/sikouhjw/gdutthesis/releases/latest
麻烦有空的各位帮我测试下以上两个版本有没有什么问题

还有收集下对模板的建议，比如：添加定理类环境等

**Comment by House-Leo** [comment]
测试了overleaf的版本，各段行距好像没有设置好，可以去知网搜搜看每年的广工优秀论文格式。
overleaf编译出来的：
![image](https://user-images.githubusercontent.com/47940711/133363879-24077e9e-77e5-4ce0-ac2d-92fc1060a9e9.png)
我知网上搜到的：
![image](https://user-images.githubusercontent.com/47940711/133363893-abd096d7-62a8-4b33-91ca-20b5f18df6cc.png)

另外好像没有算法框图的模版，可以考虑下添加。

突然搜到的一个latex模版，明年也要离开待7年的广工了，十分感谢您的工作！有需要帮助的可以一起改良这个模版。

**Comment by sikouhjw** [comment]
> 
> 
> 测试了overleaf的版本，各段行距好像没有设置好，可以去知网搜搜看每年的广工优秀论文格式。
> overleaf编译出来的：
> ![image](https://user-images.githubusercontent.com/47940711/133363879-24077e9e-77e5-4ce0-ac2d-92fc1060a9e9.png)
> 我知网上搜到的：
> ![image](https://user-images.githubusercontent.com/47940711/133363893-abd096d7-62a8-4b33-91ca-20b5f18df6cc.png)

首先，在规范中，规定了「行间距」，没有规定「段间距」。「段间距」是每段之间的额外间距，一般是西文中才会用，中文因为段首缩进，因此不需要。

其次，在 Word 中，由于官方「模板」设置了「对齐到网络」，导致规定所谓的「行间距」其实就是笑话，我相信大多数人都是在官方给的「模板」上修改的，因此会导致错误的结果。在 LaTeX 中也无法设置「对齐到网络」，因此我选择了规范中的行间距。

所以，结论是：优秀论文可能只是内容优秀，跟排版没有什么关系。

> 另外好像没有算法框图的模版，可以考虑下添加。

你这里的「模板」是指使用示例，我给了使用者自由的空间，可以去选择用哪个算法宏包。

因为规范没有规定，作为开发者去限定使用哪个宏包、哪个样式是不好的，而我暂时没有想法去做多个算法样式给用户选择（包括定理类环境也是如此，要么不限制，要么得开发很高自由度的选择，比较麻烦）。

> 突然搜到的一个latex模版，明年也要离开待7年的广工了，十分感谢您的工作！有需要帮助的可以一起改良这个模版。

祝毕业顺利，希望我的模板能给您带来方便。（可能得等（很长的）一段时间，官方才能钦定这个模板，无奈）

**Comment by House-Leo** [comment]
> > 测试了overleaf的版本，各段行距好像没有设置好，可以去知网搜搜看每年的广工优秀论文格式。
> > overleaf编译出来的：
> > ![image](https://user-images.githubusercontent.com/47940711/133363879-24077e9e-77e5-4ce0-ac2d-92fc1060a9e9.png)
> > 我知网上搜到的：
> > ![image](https://user-images.githubusercontent.com/47940711/133363893-abd096d7-62a8-4b33-91ca-20b5f18df6cc.png)
> 
> 首先，在规范中，规定了「行间距」，没有规定「段间距」。「段间距」是每段之间的额外间距，一般是西文中才会用，中文因为段首缩进，因此不需要。
> 
> 其次，在 Word 中，由于官方「模板」设置了「对齐到网络」，导致规定所谓的「行间距」其实就是笑话，我相信大多数人都是在官方给的「模板」上修改的，因此会导致错误的结果。在 LaTeX 中也无法设置「对齐到网络」，因此我选择了规范中的行间距。
> 
> 所以，结论是：优秀论文可能只是内容优秀，跟排版没有什么关系。
> 
> > 另外好像没有算法框图的模版，可以考虑下添加。
> 
> 你这里的「模板」是指使用示例，我给了使用者自由的空间，可以去选择用哪个算法宏包。
> 
> 因为规范没有规定，作为开发者去限定使用哪个宏包、哪个样式是不好的，而我暂时没有想法去做多个算法样式给用户选择（包括定理类环境也是如此，要么不限制，要么得开发很高自由度的选择，比较麻烦）。
> 
> > 突然搜到的一个latex模版，明年也要离开待7年的广工了，十分感谢您的工作！有需要帮助的可以一起改良这个模版。
> 
> 祝毕业顺利，希望我的模板能给您带来方便。（可能得等（很长的）一段时间，官方才能钦定这个模板，无奈）

谢谢回复。

另外建议可以去掉彩色封面，电子版提交的时候我记得是不用带这个的（包括我本科在广工读也是），彩色封面是最后提交实体版的时候，学校统一拿去打印封装的。

**Comment by sikouhjw** [comment]
> 另外建议可以去掉彩色封面，电子版提交的时候我记得是不用带这个的（包括我本科在广工读也是），彩色封面是最后提交实体版的时候，学校统一拿去打印封装的。

后面再提供选项吧，使用者可以指定输出或不输出封面、扉页。

**Comment by E-beer** [comment]
请问有本科生版的吗

**Comment by sikouhjw** [comment]
> 请问有本科生版的吗

我说一下目前的情况：我的导师在向研究生院推荐我的模板，然而院长并不买单。
如果学校 / 学院承认我的模板的合法性，将其认定为「官方认证」的模板，我会考虑制作本科生版。
前方困难重重。

**Comment by E-beer** [comment]
好的谢谢，共勉

**Comment by sikouhjw** [comment]
> 由于 Overleaf 还不支持 texlive 2021，我就改了一些设置以适应 texlive2020 Overleaf 版：https://cn.overleaf.com/2387673938nynfqfjzhvqt texlive 2021 版：https://github.com/sikouhjw/gdutthesis/releases/latest 麻烦有空的各位帮我测试下以上两个版本有没有什么问题

现在 Overleaf 已经有 texlive 2021 了，https://github.com/sikouhjw/gdutthesis/releases/tag/v0.1b 在上面编译成功，后面找时间上传到 Overleaf 上，打开模板就可以用了。

**Comment by ruifenggong** [comment]
现在自动化学院推的latex模板是这个仓库的吗

**Comment by sikouhjw** [comment]
> 
> 
> 现在自动化学院推的latex模板是这个仓库的吗

是的

**Comment by sikouhjw** [comment]
最新消息：《[研究生学位论文撰写规范宣讲暨LaTeX模板发布会](https://news.gdut.edu.cn/viewarticle.aspx?articleid=161298)》。
![图片](https://user-images.githubusercontent.com/49770925/141953981-17879b1b-5472-4459-9f1e-b9f45ab2f28d.png)
目前已被自动化学院推广，我为维护专员。

**Comment by sikouhjw** [comment]
基于模板推广进展，此 issue 关闭，有新建议／意见请去开新 issue 或发邮件。

**Comment by E-beer** [comment]
Thank you for your email.If you message requires a reply,I will respond immediately.Best wishes,
